.eslintrc.json

```diff
@@ -10,7 +10,8 @@
 		"@typescript-eslint/init-declarations": 0,
 		"no-underscore-dangle": 0,
 		"no-extra-parens": 0,
-		"lines-around-comment": 0
+		"lines-around-comment": 0,
+		"valid-jsdoc": 0
```

Comment:
This should be reverted - we had `valid-jsdoc` enabled as it still serves useful for ensuring TypeScript functions are documented properly.